tests/models/test_dag.py

```diff
@@ -606,6 +606,27 @@ def test_cycle(self):
         with self.assertRaises(AirflowDagCycleException):
             dag.test_cycle()
 
+    def test_long_dag_parsing(self):
+        """
+
+        Ensure DAG Parsing does not break on long DAGs
+        :return:
+        """
+        dag = DAG(
+            'dag',
+            start_date=DEFAULT_DATE,
+            default_args={'owner': 'owner1'})
+
+        # E-> A -> B -> F -> A
+        #       -> C -> F
+        with dag:
+            last = dag
+            for i in range(1,10000):
```

Comment:
```suggestion
            for i in range(1, 10000):
```

To keep pylint happy